Sources/MessagingInApp/Views/UIKitInline.swift

```diff
@@ -35,22 +35,15 @@ public class InAppMessageView: UIView, GistInlineInAppMessageViewDelegate {
     // Delegate to handle custom action button tap.
     public weak var onActionDelegate: InAppMessageViewActionDelegate?
 
-    // When a fetch request is performed, it's an async operation to have the inline View notified about this fetch and the inline View processing the fetch.
-    // There is currently no easy way to know when the inline View has finished processing the fetch.
-    // This listener is a hack for our automated tests to know when the inline View has finished processing the fetch.
-    //
-    // See linear ticket MBL-427 to learn more about this limitation in our tests.
-    var refreshViewListener: (() -> Void)?
-
     var runningHeightChangeAnimation: UIViewPropertyAnimator?
     var runningCrossFadeAnimation: UIViewPropertyAnimator?
 
     var messageRenderingLoadingView: UIView? {
         subviews.first { $0 is UIActivityIndicatorView }
     }
 
-    var inAppMessageView: UIView? {
-        subviews.first { $0 is GistInlineInAppMessageView }
+    var inAppMessageView: GistInlineInAppMessageView? {
```

Comment:
Modified this property to have strong typing to `GistInlineInAppMessageView` instead of generic `UIView`. Makes writing tests easier.